backend/matching-service/app/src/matching/matching.gateway.ts

```diff
@@ -17,8 +22,9 @@ export class MatchingGateway {
             ? {
                 ["api-gateway-auth-secret"]:
                   process.env.API_GATEWAY_AUTH_SECRET,
+                "Content-Type": "application/json",
               }
-            : {}),
+            : { "Content-Type": "application/json" }),
```

Comment:
nit: can abstract out this content-type header since its used in both paths

Reply:
a5ec34d5c2f23122dbd1b049e506d11f08a8297f